### PR TITLE
Update language feature status

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

File: docs/Language Feature Status.md [hunk 1]
```diff
@@ -10,11 +10,12 @@ efforts behind them.
 
 | Feature | Branch | State | Developers | Reviewer | LDM Champ |
 | ------- | ------ | ----- | ---------- | -------- | --------- |
-| [Ranges](https://github.com/dotnet/roslyn/blob/features/range/docs/features/range.md) | [range](https://github.com/dotnet/roslyn/tree/features/range) | Prototype | [vsadov](https://github.com/vsadov) | [agocke](https://github.com/agocke) | [jaredpar](https://github.com/jaredpar) |
+| [Ranges](https://github.com/dotnet/roslyn/blob/features/range/docs/features/range.md) | [range](https://github.com/dotnet/roslyn/tree/features/range) | Prototype | [vsadov](https://github.com/vsadov) | [OmarTawfik](https://github.com/OmarTawfik) | [jaredpar](https://github.com/jaredpar) |
```
Comment (sharwell) [inline docs/Language Feature Status.md]: ❔  Can you not use the `@OmarTawfik` syntax here?

File: docs/Language Feature Status.md [hunk 2]
```diff
@@ -10,11 +10,12 @@ efforts behind them.
...
 | [Support == and != for tuples](https://github.com/dotnet/csharplang/issues/190) | | In Progress | [jcouv](https://github.com/jcouv) | | [jcouv](https://github.com/jcouv) |
-| strongname | [strongname](https://github.com/dotnet/roslyn/tree/features/strongname) | In Progress/Backlog | | | [jaredpar](https://github.com/jaredpar) |
+| strongname | [strongname](https://github.com/dotnet/roslyn/tree/features/strongname) | Merged | [tyoverby](https://github.com/tyoverby) | [agocke](https://github.com/agocke) | [jaredpar](https://github.com/jaredpar) |
 | [Attribute on backing field](https://github.com/dotnet/csharplang/issues/42) | master | Merged | [jcouv](https://github.com/jcouv) | [AlekseyTs](https://github.com/AlekseyTs) | [jcouv](https://github.com/jcouv) |
```
Comment (jcouv) [inline docs/Language Feature Status.md]: Looks like I had made a mistake on the backing field feature. It's merged into `features/compiler` branch, not master.

File: docs/Language Feature Status.md (hunk 1 above)
Comment (jaredpar) [reply docs/Language Feature Status.md]: Not sure if that is supported or not and can't find a reference to say it is / isn't.

File: docs/Language Feature Status.md (hunk 2 above)
Comment (jaredpar) [reply docs/Language Feature Status.md]: Will fix that.

**Pull request merged**